src/main/java/seedu/address/logic/commands/NotifCommand.java

```diff
@@ -61,11 +65,21 @@ public void startSesChangeBodyStatus() {
      * @param model refers to the ModelManager
      */
     public void startSesChangeBodyStatusUi(Model model) throws CommandException {
+        Body body = toAdd.getBody();
+        String notifContent = "Body Id: " + body.getIdNum()
+                                + "\nName: " + body.getName()
+                                + "\nNext of Kin has been uncontactable. Please contact the police";
+
         Runnable changeUi = () -> Platform.runLater(() -> {
-            UpdateCommand up = new UpdateCommand(toAdd.getBody().getIdNum(), new UpdateBodyDescriptor(toAdd.getBody()));
+            UpdateCommand up = new UpdateCommand(body.getIdNum(), new UpdateBodyDescriptor(body));
             up.setUpdateFromNotif(true);
             try {
                 up.execute(model);
+
+                NotifWindow notifWindow = new NotifWindow();
+                notifWindow.setTitle("Contact Police!");
+                notifWindow.setContent(notifContent);
+                notifWindow.display();
             } catch (CommandException e) {
                 // todo what to throw?
             }
```

Comment:
I think you don't have to catch and throw again. You can just throw it back to the `execute(Model model)` method since all `CommandException` thrown from `execute(Model model)` methods will be handled in the higher abstraction levels.